resources/scss/_common.scss

```diff
@@ -0,0 +1,321 @@
+@import './_variables.scss';
+
+@mixin flex {
+  display: flex;
+}
+
+@mixin flex-row {
+  @include flex;
+  flex: 1;
+}
+
+@mixin flex-column {
+  @include flex;
+  flex: 1;
+  flex-direction: column;
+}
+
+@mixin flex-row-reverse {
+  @include flex;
+  flex-direction: row-reverse;
+}
+
+@mixin flex-column-reverse {
+  @include flex;
+  flex-direction: column-reverse;
+}
+
+@mixin break-point($point) {
+  @if $point == tablet {
+    @media only screen and (min-width: 480px) {
+      @content;
+    }
+  } @else if $point == desktop {
+    @media only screen and (min-width: 1024px) {
+      @content;
+    }
+  }
+}
+
+@mixin app-header-padding-changes {
```

Comment:
Why is this a mixin, can be a placeholder type

Reply:
Because This mixin includes the media query break-point and scss is not allowing to extend placeholder type with media query break point.